src/providers/hana/qgshanaconnectionstringbuilder.h

```diff
@@ -25,43 +25,43 @@ class QgsHanaConnectionStringBuilder
     QgsHanaConnectionStringBuilder() = default;
     explicit QgsHanaConnectionStringBuilder( const QgsDataSourceUri &uri );
 
-    const QString &getDriver() const { return mDriver; }
+    const QString &driver() const { return mDriver; }
```

Comment:
Returning by (const) ref is potentially dangerous if the underlying object goes away and needs careful usage on caller side to copy when required. Copying QString is very cheap (it's implicitly shared).

--> If in doubt, returning `QString` instead of `const QString &` is safer, for (very) performance critical code, returning by const ref might be better

Reply:
Done in 67b897c0.